packages/hydrogen/package.json

```diff
@@ -86,8 +86,8 @@
     "vitest": "^1.0.4"
   },
   "peerDependencies": {
-    "@remix-run/react": "^2.1.0",
-    "@remix-run/server-runtime": "^2.1.0",
+    "@remix-run/react": "^2.10.0",
+    "@remix-run/server-runtime": "^2.10.0",
```

Comment:
This is a peer dep so we can keep the version low like before, right?

Reply:
I see you had to dedup Remix deps and updated the peer deps to do so. I think we can keep this peer dependency as `^2.1.0` and still dedup the dependencies because it's not a requirement, `npm dedup` should still work.

Reply:
it works this time!